### PR TITLE
Use `manualTests.json` for manual test blacklisting in yaml tests

### PR DESCRIPTION
**Pull request by andy31415**
Update manual tests logic from previous `whitelist` to a `blacklist` (i.e. new tests will automatically be run):

- update from whitelist to blacklist
- Update the blacklist logic to load manualTests.json to populate a large number of manual tests

### CODE REVIEW

File: scripts/tests/chiptest/__init__.py [hunk 1]
```diff
@@ -27,6 +30,179 @@
...
+
+
+def _GetBlackListedTests() -> Set[BlackListedTest]:
+    blacklist = set()
+
+    # Flaky tests? these seem to pass though
+    blacklist.add(BlackListedTest(yaml="DL_Schedules.yaml", reason="Flaky"))
+    blacklist.add(BlackListedTest(yaml="Test_TC_LVL_2_1.yaml", reason="Flaky"))
+
+    # Examples. Should these be blacklisted?
```
Comment (tehampson) [inline scripts/tests/chiptest/__init__.py:60]: They should be prevented from running. Although for CI we are supposed to be getting the list of tests from src/app/tests/suites/ciTests.json which I am assuming will happen in a follow up

File: scripts/tests/chiptest/__init__.py [hunk 2]
```diff
@@ -27,6 +30,179 @@
...
+
+    # Examples. Should these be manual?
+    manualtests.add(ManualTest(yaml="Config_Example.yaml", reason="Example"))
+    manualtests.add(ManualTest(yaml="Config_Variables_Example.yaml", reason="Example"))
+    manualtests.add(ManualTest(yaml="PICS_Example.yaml", reason="Example"))
+    manualtests.add(ManualTest(yaml="Response_Example.yaml", reason="Example"))
+    manualtests.add(ManualTest(yaml="Test_Example.yaml", reason="Example"))
+
+    # Flagged as manual from: src/app/tests/suites/manualTests.json
+    for item in _LoadManualTestsJson(os.path.join(_YAML_TEST_SUITE_PATH, "manualTests.json")):
```
Comment (tehampson) [inline scripts/tests/chiptest/__init__.py:68]: Suggest moving `_LoadManualTestsJson` this to the top with all other manualtests.add after with a comment saying that these are not actually manual tests, but just something we are using in the interim to stop these tests from running in CI until we work our the issues

**Review by tehampson — APPROVED** [review]

File: scripts/tests/chiptest/__init__.py (hunk 1 above)
Comment (andy31415) [reply scripts/tests/chiptest/__init__.py]: I added a comment for now. ciTests works, however I am concerned about orphaned tests: what if a yaml is neither in ciTests nor in manualTests (like these)? Why do we have them then?

File: scripts/tests/chiptest/__init__.py (hunk 2 above)
Comment (andy31415) [reply scripts/tests/chiptest/__init__.py]: Moved and added comment.

File: scripts/tests/chiptest/__init__.py [hunk 3]
```diff
@@ -27,6 +30,164 @@
...
+    manualtests.add(ManualTest(
+        yaml="Test_TC_DGWIFI_3_1_Simulated.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_DRLK_2_2.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_DRLK_2_3.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_DRLK_2_5.yaml", reason="TODO"))
+    manualtests.add(ManualTest(yaml="Test_TC_DRLK_2_7.yaml", reason="TODO"))
+    manualtests.add(ManualTest(
+        yaml="Test_TC_FLABEL_3_1_Simulated.yaml", reason="TODO"))
+    manualtests.add(ManualTest(
+        yaml="Test_TC_FLW_3_1_Simulated.yaml", reason="TODO"))
```
Comment (vivien-apple) [inline scripts/tests/chiptest/__init__.py:102]: `*_Simulated` tests are not manual tests. Those are tests used by the placeholder apps.
It is unclear that you will be able to run those using `chip-repl` currently.

**Review by vivien-apple — APPROVED** [review]

**Pull request merged**